File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.common.action_chains import ActionChains


service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:    
    actions = ActionChains(driver)   
    driver.get("https://www.training-support.net/selenium/input-events")   
    print("Page title is: ", driver.title)

    ele=driver.find_element(By.XPATH,"//div[@class='active']")    
    actions.click(ele).pause(2).perform()   
    active_side = driver.find_element(By.CLASS_NAME, "active")    
    print("left click: ", active_side.text)

    actions.double_click(ele).pause(2).perform()
    active_side = driver.find_element(By.CLASS_NAME, "active")
    print("Double click: ", active_side.text)

    actions.context_click(ele).pause(2).perform()
    active_side = driver.find_element(By.CLASS_NAME, "active")
    print("Right click: ", active_side.text)

    driver.close


    
    

File: Selenium/Activities/Activity5.java
package example;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity5 {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		driver.get("https://training-support.net/webelements/dynamic-controls");
		System.out.println("Title of the page is: " + driver.getTitle());
		
		WebElement checkbox=driver.findElement(By.id("checkbox"));
		System.out.println("Checkbox is displayed: " + checkbox.isDisplayed());
		driver.findElement(By.xpath("//button[text()='Toggle Checkbox']")).click();
		System.out.println("Checkbox is displayed: " + checkbox.isDisplayed());
		driver.quit();
		

	}

}
